lib/OpenQA/WebAPI/Controller/Admin/Influxdb.pm

```diff
@@ -107,11 +107,15 @@ sub jobs {
 sub minion {
     my $self = shift;
 
-    my $stats = $self->app->minion->stats;
-    my $jobs  = {
+    my $stats           = $self->app->minion->stats;
+    my $block_list      = $self->app->config->{global}->{influxdb_minion_job_blocklist} || '';
+    my @filter_task     = split(/\s+/, $block_list);
```

Comment:
I would do the splitting when reading the settings and not every time the route is accessed.

Reply:
Put influxdb_minion to its own section, also do the splitting when reading settings.